uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup.html

```diff
@@ -386,6 +386,12 @@ <h4 class="modal-title">
         ng-click="$ctrl.cancel()">
         Back
       </button>
+      <a
+        class="text-danger pull-left pointer chr_leave-request-modal__delete"
```

Comment:
I had to remove the href=`#`, because it was messing the routing and firing the digest cycle and because of that infinite digest cycle error was coming. 
`pointer` is added to get the hand cursor.